Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -37,14 +37,24 @@ static std::unordered_map<std::string, std::string> s_textureMap;
 static std::unordered_map<std::string, std::shared_ptr<HiresTexture>> s_textureCache;
 static std::mutex s_textureCacheMutex;
 static std::mutex s_textureCacheAquireMutex;  // for high priority access
-static Common::Flag s_textureCacheAbortLoading;
+static Common::Flag s_texturePrefetcher_AbortLoading;
 static bool s_check_native_format;
 static bool s_check_new_format;
 
 static std::thread s_prefetcher;
 
 static const std::string s_format_prefix = "tex1_";
 
+// Prefetching counters
+static u32 FileType_Counter_DDS_Amount;
+static u32 FileType_Counter_PNG_Amount;
+static u32 FileType_Counter_BMP_Amount;
+static u32 FileType_Counter_TGA_Amount;  // Common for TGA and TGB variants
+static u32 FileType_Counter_JPG_Amount;  // Common for JPG, JPEG and JFIF variants
+static u32 FileType_Counter_HDR_Amount;
+static u32 FileType_Counter_Total_Amount;
+static std::string FileType_Counter_CurrentType;
```

Comment:
I think i meant it that it could be used in future in other places, this is still kinda evolving, hence "various", I will put it down in the end if I or others don't come up with ideas that will use it.